internal/plugins/helm/v1/scaffolds/init.go

```diff
@@ -94,10 +94,10 @@ func (s *initScaffolder) scaffold() error {
 		&rbac.ManagerRole{},
 		&rbac.ManagerRoleBinding{},
 		&manager.Kustomization{},
-		&manager.Manager{Image: imageName},
+		&manager.Config{Image: imageName},
 		&prometheus.Kustomization{},
 		&prometheus.ServiceMonitor{},
-		&kdefault.AuthProxyPatch{},
+		&kdefault.ManagerAuthProxyPatch{},
```

Comment:
these changes are because of the alignment to keep the file such as upstream.
just replace the names